test/conf/logback-dtest.xml

```diff
@@ -34,7 +34,7 @@
 
   <appender name="INSTANCESTDERR" target="System.err" class="ch.qos.logback.core.ConsoleAppender">
     <encoder>
-      <pattern>%-5level %date{HH:mm:ss,SSS} %msg%n</pattern>
+      <pattern>%-5level %date{"HH:mm:ss,SSS"} %msg%n</pattern>
```

Comment:
the original one is an invalid pattern, probably since logback 1.3
https://logback.qos.ch/manual/layouts.html#date